### PR TITLE
MEGRE dependecies: Set Ready translations

### PR DESCRIPTION
**Pull request by 3d-gussner**
Don't merge before #4429

Please ignore during review the /Firmware/*.* files these are part of #4429. I hope splitting the PR in two parts prevents conflicts with other existing PRs.

- :czech_republic: :heavy_check_mark: from PrusaConnect
- :de: :heavy_check_mark: from PrusaConnect
- :es: :heavy_check_mark: from PrusaConnect
- :fr:  :heavy_check_mark: from PrusaConnect
- :it: :heavy_check_mark: from PrusaConnect had to shorten it to fit
- :poland: :heavy_check_mark: from PrusaConnect
- :netherlands: :heavy_check_mark: translated by @3d-gussner. Please check @vintagepc
- :romania: :heavy_check_mark: Please check @leptun and @Hauzman Waiting for response of the rest to commit 
- :hungary: Please check @@AttilaSVK and @Hauzman 
- :croatia: Please check @Prime1910
- :slovakia: :heavy_check_mark: Please check @ingbrzy and @shatter136
- :sweden: : Please check @Painkiller56
- :norway: :heavy_check_mark: Please check @OS-kar and @trondkla Approved

### CODE REVIEW

**Comment by trondkla** [comment]
Norwegian is Good

File: lang/po/Firmware_ro.po [hunk 1]
```diff
@@ -2578,6 +2578,11 @@ msgstr "Sensibilitate"
 msgid "Mesh bed leveling failed. Please run Z calibration."
 msgstr "Nivelarea patului a eșuat. Rulează Calibrare Z."
 
+#. MSG_SET_READY c=18
+#: ../../Firmware/messages.cpp:104 ../../Firmware/ultralcd.cpp:5204
+msgid "Set Ready"
+msgstr "Set pregătit"
```
Comment (leptun) [inline lang/po/Firmware_ro.po:2584]: This is wrong. The translation used the meaning of "set" as in "a set of items", which is wrong.
"Printer pregătit" might be better in this context, but it's still not that great.

**Review by vintagepc — APPROVED** [review]

**Review by shatter136 — CHANGES_REQUESTED** [review]
I would not use "Pripravte sa" for "Set ready". It seems it means something like "Set printer as ready"  Is that correct? In such case the best would be "Nastaviť ako pripravené". However, I see Czech translation is "Připravit". So if it must be shorter message, Slovak could be "Pripraviť" or "Pripravené"

File: lang/po/Firmware_sk.po [hunk 2]
```diff
@@ -2560,6 +2560,11 @@ msgstr "Citlivosť"
 msgid "Mesh bed leveling failed. Please run Z calibration."
 msgstr "Vyrovnanie platne zlyhalo. Spustite kalibráciu Z."
 
+#. MSG_SET_READY c=18
+#: ../../Firmware/messages.cpp:104 ../../Firmware/ultralcd.cpp:5204
+msgid "Set Ready"
+msgstr "Pripravte sa"
```
Comment (ingbrzy) [inline lang/po/Firmware_sk.po:2566]: Should be "Pripravené"

File: Firmware/cmdqueue.cpp [hunk 3]
```diff
@@ -479,9 +479,10 @@ void get_command()
             allow_when_stopped = true;
 
         // Handle the USB timer
-        if ((*cmd_start == 'G') && !(IS_SD_PRINTING))
+        if ((*cmd_start == 'G') && !(IS_SD_PRINTING)) {
             usb_timer.start();
-
+            printer_state = 3; //set printer state busy printing to hide LCD menu and report correctly M862.7 Q while USB printing
```
Comment (gudnimg) [inline Firmware/cmdqueue.cpp:484]: This conflicts with https://github.com/prusa3d/Prusa-Firmware/pull/4429

File: Firmware/Marlin_main.cpp [hunk 4]
```diff
@@ -7957,6 +7968,14 @@ SERIAL_PROTOCOLPGM("\n\n");
...
                          SERIAL_PROTOCOLLN(GCODE_LEVEL);
                     break;
+               case ClPrintChecking::_Features:  // ~ .6
+                    break;
+               case ClPrintChecking::_PrinterState: // ~.7
+                    if(code_seen('P'))
+                         printer_state=code_value_uint8();
+                    else if(code_seen('Q'))
+                         SERIAL_PROTOCOLLN((int)printer_state);
+                    break;
```
Comment (gudnimg) [inline Firmware/Marlin_main.cpp:7978]: I think this should be in https://github.com/prusa3d/Prusa-Firmware/pull/4429 it's missing the `~.7`?

**Review by gudnimg — CHANGES_REQUESTED** [review]
This PR seems to conflict with https://github.com/prusa3d/Prusa-Firmware/pull/4429 for non-translation specific changes.

**Comment by 3d-gussner** [comment]
Add all translation to https://github.com/prusa3d/Prusa-Firmware/pull/4429

**Pull request closed without merging**